diagnostic.md

```diff
@@ -5,74 +5,81 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+To be able to store data
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+Controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+Views are mostly front end interactions with our app, we are looking mostly at the back end when it comes to the MVC
 ```
```

Comment:
Yes but this is because the SPA standard is so much better than the traditional view of MVC's